index.js

```diff
@@ -1,20 +1,748 @@
+  "",
+  ["knife", "fruit", "gallon jug"],
+  ["carpet"],
+  ""
+);
+
+let cargoRoom = new Room(
+  "Entering this dark hole seems like a terrible choice.\nClimbing down is difficult but you manage.\nIt is almost pitch black down here and that smell is even worse.\nIf only you had a light.",
+  "",
+  ["special game winner"],
+  ["monster", "storage containers", "body parts"]
+);
+
+let cockpitRoom = new Room(
+  "The windshields have been smashed out.\nThere seems to be a ladder attached to the nose of the plane.\nA crackling noise emanates from a radio on the floor.",
+  "",
+  [],
+  ["radio", "electrical plane controls"]
+);
+
+let caveEntrance = new Room(
+  "It seems like there was a huge earthquake here and the ladder led you to the entrance of some caves.\nPiles of huge rocks are tumbled all around.\nYou see a possible path up ahead to a cave.",
+  "",
+  [],
+  ["rubble"]
+);
+
+let caveOne = new Room(
+  "The cave is medium sized and you no longer have to lower your head.\nThere's a faint glow emanating from another passageway.\nBathing the pitch black space in a eerie bluish light."
+);
+
+let caveTwo = new Room(
+  "Suddenly your vision comes into focus and you can't believe your eyes. Inside of a large cavern a ring of hooded figures are on there knees surrounding a glowing orb.\nIt looks like they worship it.\nYour sense of adventure compels you to go forward. But you're also really scared, maybe you should go back."
+);
+
+//Global Variables//
+let prompt = ">_";
+let answer;
+
+//Begin Game
 start();
 
+//Every async function we have set up operates in almost the same way => SO we will offer comments for this first room and additional comments for a few unique sections of code we have in later rooms
+//We print out the list of optional commands the user may input
+//The function enters a while loop that has a large conditional tree that will keep looping if the input is incorrect
+//or will return a value or offer a side effect if the input is a valid command
 async function start() {
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
-There is a door here. A keypad sits on the handle.
-On the door is a handwritten sign.`;
-  let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
-  process.exit();
+  console.log(
+    `Welcome:\nCommands: open door | read | look around | take | display inventory`
+  );
+  answer = await ask(prompt);
+  //Room interaction block
+  while (answer.trim() !== true) {
+    if (answer.trim() === "look around") {
+      //calls method in room class that prints room introduction
+      console.log(roomOne.read());
+    } else if (answer.trim().includes("read")) {
+      //provides information that may be written on an item or sign
+      console.log(`The ${roomOne.props} has a passcode on it - 1234`); //pass code to open door
+    }
```

Comment:
I noticed when I scanned your program that you guys used/called `.trim()` 51 times! It may be worth creating a "Sanitize" function for future programs and calling that function immediately on inputs that are taken in. In this example here, when the variable `answer` is taken in, you could trim/lowercase/uppercase/etc all at once, and set that as the variable for all your comparisons and guard clauses.